client/sites/components/sites-dashboard.tsx

```diff
@@ -193,8 +206,8 @@ const SitesDashboard = ( {
 				},
 			},
 		},
-	} as DataViewsState;
-	const [ dataViewsState, setDataViewsState ] = useState< DataViewsState >( defaultDataViewsState );
+	} as View;
```

Comment:
I'd like to find a way to remove this assertion and type it as a `View` directly. This also relates to the ts-expect-error's below.